impl/src/main/java/com/okta/authn/sdk/impl/resource/DefaultFactorEnrollRequest.java

```diff
@@ -22,17 +22,38 @@
 import com.okta.sdk.impl.resource.Property;
 import com.okta.sdk.impl.resource.ResourceReference;
 import com.okta.sdk.impl.resource.StringProperty;
-import com.okta.sdk.resource.user.factor.FactorProfile;
+import com.okta.sdk.resource.user.factor.CallUserFactorProfile;
+import com.okta.sdk.resource.user.factor.EmailUserFactorProfile;
 import com.okta.sdk.resource.user.factor.FactorProvider;
 import com.okta.sdk.resource.user.factor.FactorType;
+import com.okta.sdk.resource.user.factor.HardwareUserFactorProfile;
+import com.okta.sdk.resource.user.factor.PushUserFactorProfile;
+import com.okta.sdk.resource.user.factor.SecurityQuestionUserFactorProfile;
+import com.okta.sdk.resource.user.factor.SmsUserFactorProfile;
+import com.okta.sdk.resource.user.factor.TokenUserFactorProfile;
+import com.okta.sdk.resource.user.factor.TotpUserFactorProfile;
+import com.okta.sdk.resource.user.factor.U2fUserFactorProfile;
+import com.okta.sdk.resource.user.factor.UserFactorProfile;
+import com.okta.sdk.resource.user.factor.WebAuthnUserFactorProfile;
+import com.okta.sdk.resource.user.factor.WebUserFactorProfile;
 
 import java.util.Map;
 
 public class DefaultFactorEnrollRequest extends AbstractResource implements FactorEnrollRequest {
 
     private static final StringProperty STATE_TOKEN_PROPERTY = new StringProperty("stateToken");
     private static final EnumProperty<FactorType> FACTOR_TYPE_PROPERTY = new EnumProperty<>("factorType", FactorType.class);
-    private static final ResourceReference<FactorProfile> FACTOR_PROFILE_PROPERTY = new ResourceReference<>("profile", FactorProfile.class, true);
```

Comment:
This change can probably be reverted when using the _new_ `UserFactorProfile`

Reply:
sure yes. done.